### DISCUSSION TITLE
get-dbalogin - remote server and WARNING: [09:46:23][Copy-DbaLogin] Failure | Access is denied

### DISCUSSION THREAD

**Discussion by kirkhughes1980** [dataplat/dbatools#8431] [Q&A]
Good Morning All. 

I'm trying to build on my skillset and learn DBATOOLS and use it as my go to scripting for admin tasks but whereas some have worked perfectly, this am, I've hit an issue.

I'm trying to get-dbalogin from one server and copy-dbalogin to another. Tried multiple ways of doing it:

 get-dbalogin -SqlInstance  'X' | Copy-DbaLogin -Destination 'Y' or Copy-DbaLogin -Source 'X' -Destination 'Y' but get the error message [Copy-DbaLogin] Failure | Access is denied

I've tried to break it down and it seems that even get-dbalogin will give the message for the -destination server. -source returns rows.

I'm local admin and sysadmin on both servers. Only thing I can think is servers are on different subnets.

Any thoughts welcome

**Comment by potatoqualitee** [comment]
hey @kirkhughes1980 sorry for the delayed response. Can you throw a `-Verbose` on that and paste it here?
